internal/build/docker.go

```diff
@@ -378,28 +261,19 @@ func tarContextAndUpdateDf(ctx context.Context, df Dockerfile, paths []pathMappi
 	}()
 
 	// TODO: maybe write our own tarWriter struct with methods on it, so it's clearer that we're modifying the tar writer in place
-	dnePaths, err := archiveIfExists(ctx, tw, paths)
+	err := archiveIfExists(ctx, tw, paths)
```

Comment:
since this is no longer responsible for returning missing paths, can rename if you want

Reply:
`archivePaths`? `makeContextArchive`? `tarContext`?

Reply:
changed to archivePaths